modules/videoio/src/cap_gstreamer.cpp

```diff
@@ -1936,15 +1954,20 @@ bool GStreamerCapture::setProperty(int propId, double value)
         return false;
     }
 
+    gint64 newPos = 0;
+
     bool wasPlaying = this->isPipelinePlaying();
-    if (wasPlaying)
+    if (wasPlaying && propId == CV_CAP_PROP_POS_FRAMES)
```

Comment:
Shouldn't we stop pipeline for POS_MSEC property? Or POS_AVI_RATIO? Or WIDTH/HEIGHT?

Reply:
I checked earlier and stopping-restarting was an old implementation in the sources. When I checked GStreamer docs, I found no reference to why the pipeline should be stopped for any seeking or other behavior.
The latest version stops the pipeline for fps-image width and image height changes. I also found mov and mp4 files where a temporal pipeline stop is needed. It is added with comment.
POS_AVI_RATIO: It is basically not supported in GStreamer. I added a new OpenCV warning and comment to the source code about it.
CV_CAP_PROP_POS_FRAMES: I could not make it work.